Grbl_Esp32/src/Jog.cpp

```diff
@@ -37,8 +37,13 @@ Error jog_execute(plan_line_data_t* pl_data, parser_block_t* gc_block) {
             return Error::TravelExceeded;
         }
     }
-    // Valid jog command. Plan, set state, and execute.
+// Valid jog command. Plan, set state, and execute.
+#ifndef USE_KINEMATICS
```

Comment:
This ifdef could be eliminated by just calling inverse_kinematics(), and then making inverse_kinematics() into a weak symbol that binds to mc_line()

Reply:
I was looking at that for other areas of the kinematics. I put it on hold. When I get time, I want to look at making the various kinematics follow a more standardized template. I feel like I am reinventing each time.